wireapp/wire-android-reloaded
# fix(calling): Self video stays paused when moving the app into foreground in video call (AR-1825)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1825" title="AR-1825" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />AR-1825</a>  1:1 calling - Take app to background and foreground again, video stays paused
  </td></table>
  <br />
 




















----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

Self video stays paused when moving the app into foreground in video call

### Causes (Optional)

We are not enabling the video again when the app return into foreground

### Solutions

Check if the app is back in the ongoing call screen and there is an established call then we turn on video


Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

Start a call
Turn on your video
Put app in background
Return it into foreground from task manager
Video should be turned on again

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
